src/ir.h

```diff
@@ -133,7 +133,6 @@ struct FuncSignature {
 };
 
 struct FuncType {
-  FuncType() = default;
```

Comment:
hm, not sure how I feel about this change. I don't think these should always have names.

Reply:
But all the callsites to provide a name.   No point in having non-used constructor is there?

Reply:
Guess so.